### PR TITLE
Migrating the project membership pages from ember to vue

### PR DESCRIPTION
**Pull request by codyrancher**
rancher/dashboard#2673
project-membership

### CODE REVIEW

**Comment by codyrancher** [comment]
Waiting on https://github.com/rancher/dashboard/issues/2501

**Review by westlywright — COMMENTED** [review]
Couple small things in the code.

**Comment by westlywright** [comment]
When I pulled the PR down and tested I found one issue. After saving the new project member in the local cluster the redirect fails with "Page Not Found". But it looks like I was redirected back to `https://localhost:8005/` so I'm unsure what's going on there. If you can't repro ping me on slack and we can figure it out.

File: components/SortableTable/index.vue [hunk 1]
```diff
@@ -506,11 +506,12 @@ export default {
             </template>
           </slot>
         </div>
-
         <div v-if="$slots['header-middle'] && $slots['header-middle'].length" class="middle">
           <slot name="header-middle" />
         </div>
 
+        <slot name="headerMiddlre" />
```
[the remark this answers is not in the document]
Comment (codyrancher) [reply components/SortableTable/index.vue:513]: Removing. It was used before I switched over to an ArrayList.

File: edit/management.cattle.io.project.vue [hunk 2]
```diff
@@ -67,25 +159,91 @@ export default {
...
+      <Tab name="members" :label="t('project.members')" :weight="10">
+        <ArrayList
+          v-model="projectBindings"
+          :show-header="true"
+        >
+          <template #column-headers>
+            <div class="box mb-0">
+              <div class="column-headers row">
+                <div class="col span-6">
+                  <label class="text-label">User</label>
```
[the remark this answers is not in the document]
Comment (codyrancher) [reply edit/management.cattle.io.project.vue:212]: Fixed

File: edit/management.cattle.io.project.vue [hunk 3]
```diff
@@ -67,25 +159,91 @@ export default {
...
+                </div>
+                <div class="col span-6">
+                  <label class="text-label">Role</label>
```
[the remark this answers is not in the document]
Comment (codyrancher) [reply edit/management.cattle.io.project.vue:215]: Fixed

**Comment by codyrancher** [comment]
I got a ship it through slack before merging.

**Pull request merged**